docs/zh/kustomization.yaml

```diff
@@ -0,0 +1,287 @@
+# to be copied literally.  The values won't work if
+# they happen to be references to external files that
+# don't exist.
+#
+# In practice, fields with no value should simply be
+# omitted from kustomization.yaml to reduce the content
+# visible in configuration reviews.
+# ----------------------------------------------------
+# Kustomization 的 apiVersion 和 kind 
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# 为所有 resources 添加 namespace
+namespace: my-namespace
+
+# 该字段的值将添加在所有资源的名称之前
+# 例如 将资源名称 “wordpress” 变为 “alices-wordpress” 
+namePrefix: alices-
+
+# 该字段的值将添加在所有资源的名称后面
+# 例如 将资源名称 “wordpress” 变为 “wordpress-v2”
+# 如果资源类型为 ConfigMap 或 Secret ，则在哈希值之前附加后缀
+nameSuffix: -v2
+
+# 为所有资源和 selectors 增加 Labels
+commonLabels:
+  someName: someValue
+  owner: alice
+  app: bingo
+
+# 和 Labels 一样， 增加 Annotations
+# 为 key:value 键值对
+commonAnnotations:
+  oncallPager: 800-555-1212
+
+# 此列表中的每条记录都必须是一个存在的 YAML 资源描述文件
+# 这些 YAML 资源描述文件将被 kustomize 读取，修改和发布
+# 资源由文档标记（"---"）分隔
+resources:
+- some-service.yaml
+- sub-dir/some-deployment.yaml
+
+# 列表中的每个条目都将创建一个 ConfigMap （它是n个映射的生成器）
+# 下面的示例创建了两个 ConfigMaps
+# 一个具有给定文件的名称和内容
+# 另一个包含 key/value 键值对数据
+# 每个 configMapGenerator 项都可以使用 [create | replace | merge] 参数
+# 允许 overlay 从父级修改或替换现有的 configMap
+configMapGenerator:
+- name: myJavaServerProps
+  files:
+  - application.properties
+  - more.properties
+- name: myJavaServerEnvVars
+  literals:	
+  - JAVA_HOME=/opt/java/jdk
+  - JAVA_TOOL_OPTIONS=-agentlib:hprof
+
+# 此列表中的每个条目都会导致创建一个Secret资源（n个 secrets 的生成器）
+secretGenerator:
+- name: app-tls
+  files:
+    - secret/tls.cert
+    - secret/tls.key
+  type: "kubernetes.io/tls"
+- name: app-tls-namespaced
+  # 你可以给生成的 secret 定义一个 namespace ，默认为 ”default“
+  namespace: apps
+  files:
+    - tls.crt=catsecret/tls.cert
+    - tls.key=secret/tls.key
+  type: "kubernetes.io/tls"
+- name: env_file_secret
+  # 文件路径以 k=v 键值对的形式，每行一个键值对
+  envs:
+    - env.txt
+  type: Opaque
+
+# generatorOptions 修改所有 ConfigMap 和 Secret 的行为
+generatorOptions:
+  # 为所有生成的资源添加 labels
+  labels:
+    kustomize.generated.resources: somevalue
+  # 为所有生成的资源添加 annotations
+  annotations:
+    kustomize.generated.resource: somevalue
+  # disableNameSuffixHash 为 true 时将禁止默认的在名称后添加哈希值后缀的行为
+  disableNameSuffixHash: true
+
+# 此列表中的每个条目都应解析为包含 kustomization 文件的目录，否则定制将失败
+#
+# 该条目可以是指向本地目录的相对路径
+# 也可以是指向远程仓库中的目录的 URL
+# URL 应该遵循 hashicorp/go-getter 中的 URL 格式
+# https://github.com/hashicorp/go-getter#url-format
+#
+# 此字段的存在意味着此文件（您正在阅读的文件）是 _overlay_
+# 它将进一步定制这些来自 _bases_ 文件中的配置
+#
+# 典型用例：开发，演示和生产环境
+# 这些环境大部分相同但有些关键方式存在差异（镜像标签，一些服务器参数等，与公共 base 不同的配置）
+bases:
+- ../../base
+- github.com/kubernetes-sigs/kustomize/examples/multibases?ref=v1.0.6
+- github.com/Liujingfang1/mysql
+- github.com/Liujingfang1/kustomize/examples/helloWorld?ref=test-branch
+
+# 此列表中的每个条目都应可以解析为部分或完整的资源定义文件
+#
+# 这些（也可能是部分的）资源文件中的 name 必须与已经通过 `resources` 加载的 name 字段匹配
+# 或者通过 `bases` 中的 name 字段匹配
+# 这些条目将用于 _patch_（修改）已知资源
+#
+# 最好做一些小的 patches 
```

Comment:
```
推荐使用小的 patches
```

Reply:
This translation is much better than mine!
Can you speak Chinese? I mean your name looks like a Chinese name.
I encountered some confusion when translating the [glossary](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md). The role of the glossary is to disambiguate. I don't want my translation to bring more ambiguity.

Reply:
@sunny0826 Yes, I can speak Chinese. 
If you get better translation, feel free to update the doc.